docs/podman-image-exists.1.md

```diff
@@ -0,0 +1,46 @@
+% PODMAN(1) Podman Man Pages
+% Brent Baude
+% November 2018
+# NAME
+podman-image-exists- Check if an image exists in local storage
+
+# SYNOPSIS
+**podman image exists**
+[**-h**|**--help**]
+[**-q**|**--quiet**]
+IMAGE
+
+# DESCRIPTION
+**podman image exists** checks if an image exists in local storage. You may use the **ID** or **Name**
```

Comment:
Ditto 'you' comment from previous man page.

Reply:
fixed